src/blocks/disk_space.rs

```diff
@@ -88,6 +88,9 @@ pub struct DiskSpaceConfig {
     #[serde(default = "DiskSpaceConfig::default_alert")]
     pub alert: f64,
 
+    /// Show percentage
+    #[serde(default = "DiskSpaceConfig::default_show_percentage")]
+    pub show_percentage: bool,
```

Comment:
Instead of adding another configuration option, can you implement percentages as another `Unit` type?

Reply:
Making it `Unit` type removes the possibility to show both GB and percentage at the same time. Originally I meant that the percentage is just and extra option, not `Unit`type.

Reply:
I'm not quite sure what the right approach is here. This might be good for now, with the possible addition of a `percent` unit in the future so that users could print *only* the percent used, if so desired. @greshake do you have an opinion on this?

Reply:
Maybe that could be it. Make a new `Unit` type and still keep the `show_percentage` option which is active if the Unit type is not percentage.

Reply:
Your solution is fine with me. Also shouldn't require any breaking changes for existing configs